DEVELOPER_GUIDE.md

```diff
@@ -452,4 +457,4 @@ The Github workflow in [`backport.yml`](.github/workflows/backport.yml) creates
 original PR with an appropriate label `backport <backport-branch-name>` is merged to main with the backport workflow 
 run successfully on the PR. For example, if a PR on main needs to be backported to `1.x` branch, add a label 
 `backport 1.x` to the PR and make sure the backport workflow runs on the PR along with other checks. Once this PR is 
-merged to main, the workflow will create a backport PR to the `1.x` branch.
+merged to main, the workflow will create a backport PR to the `1.x` branch.
```

Comment:
What is the difference between these lines?

Reply:
Nothing intentional. I may have deleted a trailing whitespace but I'm not sure what triggered the diff.